packages/react/src/combobox-popover-content/ComboboxPopoverContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+import { PopoverContent } from "../popover-content";
+import * as styles from "./ComboboxPopoverContent.css";
+
+type ComboboxPopoverContentProps = ComponentPropsWithoutRef<
+  typeof PopoverContent
+> &
+  styles.ContentVariants;
+
+export const ComboboxPopoverContent = forwardRef<
+  HTMLDivElement,
+  ComboboxPopoverContentProps
+>(({ className, minW = "trigger", ...props }, ref) => {
+  return (
+    <PopoverContent
+      ref={ref}
+      {...styles.content({ minW }, className)}
```

Comment:
we can just forward `minW` to PopoverContent since it already supports this prop
```suggestion
      minW={minW}
```